crates/re_ui/src/section_collapsing_header.rs

```diff
@@ -1,75 +1,11 @@
-use crate::{list_item, DesignTokens, Icon, UiExt as _};
-
-/// Icon button to be used in the header of a panel.
-pub struct HeaderMenuButton<'a> {
```

Comment:
didn't I review this being moved just a few hours ago :smile:

Reply:
That pesky thing had a short—but intense—life :)